Ginger/GingerCore/Actions/ActSikuli.cs

```diff
@@ -584,5 +602,17 @@ private List<int> GetCustomResolutionValues()
 
             return lstVal;
         }
+
+        private void SetProcessAsPerVE()
+        {
+            int countSimilar = ActiveProcessWindows.Where(p => p.Contains(ProcessNameVEForSikuliOperation)).Count();
+            if (countSimilar > 0)
+            {
+                if (countSimilar == 1)
```

Comment:
please merge this two if's together and use .Any() in the if instead of creating countSimilar variable

Reply:
not using .Any() because we need a 1:1 mapping, if 2 instances of same process are open then we will face problems